### ISSUE TITLE
Examples, please!

### ISSUE THREAD

**Issue by iurisilvio** [voronind/django-awesome-avatar#9]
I had a problem with my upload because I forgot the `enctype` in my form. It is basic stuff, but can be fixed with an working example.

**Comment by Sloy** [comment]
Yep, some example would be great, because right now I can't figure out how to use this in my current form.
